src/engraving/rendering/dev/slurtielayout.cpp

```diff
@@ -1656,7 +1656,11 @@ double SlurTieLayout::noteOpticalCenterForTie(const Note* note, bool up)
         double singleSlashWidth = note->symBbox(SymId::noteheadSlashHorizontalEnds).width();
         double noteWidth = note->width();
         double center = 0.20 * note->spatium() + 0.5 * (noteWidth - singleSlashWidth);
-        return up ? note->shape().right() - center : note->shape().left() + center;
+        Shape noteShape =  note->shape();
+        noteShape.remove_if([&](ShapeElement& s) {
+            return s.item()->pos().x() + s.item()->width() > note->pos().x() + note->width();
```

Comment:
Yikes!  I was hoping it wouldn't need to be something so ugly.  There's no way to just get the note width excluding the dot more directly?

Reply:
This method is a little more foolproof and will account for elements other than dots as well (such as bends/chordlines perhaps)

Reply:
@mike-spa do you think `noteShape.removeTypes({ ElementType::NOTEDOT });` would suffice, instead?

Reply:
That's definitely a good goal; I'm just surprised there isn't a more straightforward way to get what we want, like just dealing with the notehead directly.  But given that it's a special case anyhow, I'm happy with whatever gets the job done!